server/routes/authRoutes.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+
+const db = require('../db/users')
+const token = require('../auth/token')
+const hash = require('../auth/hash')
+
+const router = express.Router()
+
+router.post('/register', register, token.issue)
+
+function register (req, res, next) {
```

Comment:
if you add `const {firstName, surName, email, hash} = req.body` after this line, you can remove `req.body` from the rest of this function.